charts/opentelemetry-collector/values.yaml

```
@@ -87,6 +87,17 @@ presets:
     #   combine_field: body
     #   source_identifier: attributes["log.file.path"]
     #   is_first_entry: body matches "^([0-9]{4}-[0-9]{2}-[0-9]{2} [0-9]{2}:[0-9]{2}:[0-9]{2}|[0-9]{2}-[0-9]{2} [0-9]{2}:[0-9]{2}:[0-9]{2})"
+    # multilineConfigs:
+    #   - namespaceName:
+    #       value: kube-system
+    #     podName:
+    #       value: app-.*
+    #       useRegex: true
+    #     containerName:
+    #       value: http
+    #     firstEntryRegex: ^[^\s].*
+    #     combineWith: ""
+    multilineConfigs: []
```

Small suggestion, to make it uniform with other parameters. Maybe would be good to add a sentence about what it is / does.
```suggestion
    multilineConfigs: []
    #   - namespaceName:
    #       value: kube-system
    #     podName:
    #       value: app-.*
    #       useRegex: true
    #     containerName:
    #       value: http
    #     firstEntryRegex: ^[^\s].*
    #     combineWith: ""
```